hardware_accelerators/nvidia-gpu-architecture.adoc

```diff
@@ -0,0 +1,106 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="nvidia-gpu-architecture"]
+= NVIDIA GPU architecture
+include::_attributes/common-attributes.adoc[]
+:context: nvidia-gpu-architecture
+
+toc::[]
+
+NVIDIA supports the use of graphics processing unit (GPU) resources on {product-title}. {product-title} is a security-focused and hardened Kubernetes platform developed and supported by Red Hat for deploying and managing Kubernetes clusters at scale. {product-title} includes enhancements to Kubernetes so that users can easily configure and use NVIDIA GPU resources to accelerate workloads.
+
+The NVIDIA GPU Operator leverages the Operator framework within {product-title} to manage the full lifecycle of NVIDIA software components required to run GPU-accelerated workloads.
```

Comment:
I would choose a more accessible word instead of "leverage". Maybe "uses" or "takes advantage of".

Reply:
changed.